compiler/msgs.nim

```diff
@@ -929,7 +927,7 @@ proc rawMessage*(msg: TMsgKind, args: openArray[string]) =
     if msg notin gNotes: return
     title = HintTitle
     color = HintColor
-    kind = HintsToStr[ord(msg) - ord(hintMin)]
+    if msg != hintUserRaw: kind = HintsToStr[ord(msg) - ord(hintMin)]
```

Comment:
`hintUserRaw` is a the very last element in the Hints sequence. It's used when you don't want to output even the "Hint: " qualification, but you still want to collect the message as a diagnostic. I used in one place to list the overloads that were considered while trying to match a concept.